sbnanaobj/StandardRecord/SRSoftwareTrigger.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file   /sbnanaobj/StandardRecord/SRSoftwareTrigger.h
+ * @brief  Defines CAF data structures to store sbnd::trigger::pmtSoftwareTrigger objects (header). 
+ * @author Lynn Tung
+ * @date   November 12, 2025 
+ *
+ */
+
+#ifndef SRSOFTWARETRIGGER_H
+#define SRSOFTWARETRIGGER_H
+
+#include "sbnanaobj/StandardRecord/SRConstants.h"
+
+namespace caf{
+  
+  /**
+  * @brief A struct to store software trigger information for SBND Data and MC
+  */
+  
+  struct SRSoftwareTrigger { 
+    int   npmts          = caf::kUninitializedInt;
+    float flash_peaktime = caf::kSignalingNaN; 
+    float flash_peakpe   = caf::kSignalingNaN;
```

Comment:
Place a comment explaining units and details. For example:

```suggestion
    int   npmts          = caf::kUninitializedInt; ///< Number of PMT pairs above threshold at trigger time.
    float flash_peaktime = caf::kSignalingNaN;     ///< Peak time of the reconstructed flash associated to the trigger [us]
    float flash_peakpe   = caf::kSignalingNaN;     ///< Peak amplitude of the reconstructed flash associated to the trigger, in photoelectron units.
```
I don't know if these descriptions are correct (that's why some should be there! 😄).
Exception: the time unit _can_ be omitted if it is the standard CAF one (microseconds). However, given that sometimes trigger-related times are specified in nanoseconds, it's still worth to be explicit.